sdks/python/apache_beam/testing/util.py

```diff
@@ -179,10 +179,11 @@ def _equal(actual):
           expected_list.remove(element)
         except ValueError:
           raise BeamAssertException(
-              'Failed assert: %r == %r' % (expected, actual))
+              'Failed assert: %r == %r, expected missing %r' %
```

Comment:
'expected missing X' may be also read as:  expected the missing element X. Consider rewording as "unexpected element" and "missing element".